nestor_api/lib/config.py

```diff
@@ -46,6 +46,16 @@ def get_app_config(app_name: str, config_path: str = Configuration.get_config_pa
     return _resolve_variables_deep(config)
 
 
+def get_applications(app_config: dict) -> list:
+    """Get applications processes (not cronjobs)"""
+    return list(filter(lambda process: process["is_cronjob"] is False, app_config["processes"]))
+
+
+def get_cronjobs(app_config: dict) -> list:
```

Comment:
Then either `get_application_cronjobs` or rename the first one ``get_processes``